modules/add-a-custom-header-and-footer.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-security-notices.adoc
+:_module-type: PROCEDURE
+[id="add-a-custom-header-and-footer_{context}"]
+= Adding a custom header and footer
+
+.Prerequisites
+
+* You need the `Config` role with `read` permission to view the custom header and footer configuration options.
```

Comment:
Might be personal preference, but I think I more often see "You must have" instead of "You need".

Same for other module, if you do change it. (No worries if you dont!)